pkg/nuclide-terminal/lib/terminal-view.js

```diff
@@ -202,6 +204,10 @@ export class TerminalView implements PtyClient {
         .observeAsStream(SCROLLBACK_CONFIG)
         .skip(1)
         .subscribe(scrollback => terminal.setOption('scrollback', scrollback)),
+      featureConfig
+        .observeAsStream(FONT_FAMILY_CONFIG)
+        .skip(1)
+        .subscribe(fontFamily => terminal.setOption('fontFamily', fontFamily)),
```

Comment:
instead of adding a new call to setOption for fontFamily (which will conflict with the existing one that sync the Atom text editor setting), could you instead rename existing 'editor.fontFamily' to be FONT_FAMILY_CONFIG (line 136)?

Then change syncAtomStyleItem (~line 400) to match the new terminal group (ideally you could also add options for the fontSize and lineHeight).

Reply:
I'm not exactly sure what syncAtomStyle does and why it is there - I've changed it to use the new Option but instead of using the existing I just set the options directly.
Using syncAtomStyle made it slow and sluggish on several occasions.
No Idea why exactly - Not that familiar with the nuclide or atom code.

Doing it like this works though.